libs/langchain/langchain/retrievers/cohere_rag_retriever.py

```diff
@@ -15,17 +15,26 @@
 
 
 def _get_docs(response: Any) -> List[Document]:
-    return [
+    docs = [
         Document(page_content=doc["snippet"], metadata=doc)
         for doc in response.generation_info["documents"]
     ]
+    docs.append(
+        Document(
+            page_content=response.message,
+            metadata={
+                "type": "model_response",
+                "citations": response.generation_info["citations"],
+                "search_results": response.generation_info["search_results"],
+                "search_queries": response.generation_info["search_queries"],
+            },
+        )
+    )
+    return docs
 
 
 class CohereRagRetriever(BaseRetriever):
-    """`ChatGPT plugin` retriever."""
-
-    top_k: int = 3
```

Comment:
not backwards compatible. probably ok since this is a pretty new integration but just want to be sure this is really worth removing